### PR TITLE
fix: Create safe layout adjustments

### LINKED ISSUE #995: [Safe Creation] Layout
## What is the feature about

Layout related tasks for the new safe creation design

## The list of requirements

- Hide sidebar
- Stay on safe creation when changing network
- Correct column sizes and spacings based on the grid

## Links

Figma: https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=51%3A5661

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Resolves #995

## How this PR fixes it

- Hides sidebar on create-safe page
- Stays on create-safe page when switching network
- Centers create-safe form and uses a fixed width container

## Screenshots
Screenshot 2022-10-28 at 10 50 26

### CODE REVIEW

File: src/components/new-safe/CardStepper/index.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export function CardStepper<StepperData>(props: TxStepperProps<StepperData>) {
   const { activeStep, onSubmit, onBack, stepData, setStep } = useCardStepper<StepperData>(props)
   const { steps } = props
   const currentStep = steps[activeStep]
-  const progress = (activeStep + 1 / steps.length) * 100
+  const progress = ((activeStep + 1) / steps.length) * 100
```
Comment (DiogoSoaress) [inline src/components/new-safe/CardStepper/index.tsx:10]: 🎖️

**Comment by DiogoSoaress** [comment]
Should we disable the continue button if you are in the wrong chain?
Screenshot 2022-10-28 at 11 10 44

File: src/components/new-safe/CreateSafe/index.tsx [hunk 2]
```diff
@@ -90,10 +86,10 @@ const CreateSafe = () => {
           </Card>
         )}
       </Grid>
-      <Grid item xs={12} md={4}>
+
+      <Grid item xs={12} md={3} mt={[3, 0]}>
```
Comment (DiogoSoaress) [inline src/components/new-safe/CreateSafe/index.tsx:90]:
```suggestion
      <Grid item xs={12} md={3} mt={[3, 3, 0]}>
```

**Review by DiogoSoaress — CHANGES_REQUESTED** [review]
👍  Just a tiny suggestion on the margin top for `md` viewport

**Comment by usame-algan** [comment]
> Should we disable the continue button if you are in the wrong chain?

Good idea, lets do it in this branch as well!

**Comment by usame-algan** [comment]
> Should we disable the continue button if you are in the wrong chain?

After discussing with @liliiaorlenko and @schmanu we decided to implement the missing wallet connection step so I suggest we add this feature there. I created a new task for it: https://github.com/safe-global/web-core/issues/1003

**Review by DiogoSoaress — APPROVED** [review]
Nice!

**Pull request merged**